### PR TITLE
Fix the KubeletConfig Document

### PR DESCRIPTION
**Pull request by Vincent056**
Updates `KubeletConfig` documents

### CODE REVIEW

**Review by jhrozek — COMMENTED** [review]
If we're improving the documentation, can we also standardize on using `KubeletConfig` instead of `KubletConfig`? Otherwise looks good.

File: doc/kubeletConfig-remediations.md [hunk 1]
```diff
@@ -11,41 +11,45 @@ We want to accomplish:
 ## Add CaC `KubletConfig` remediation content 
 In CaC content, we are going to add `KubletConfig` remediation besides `MachineConfig` that we currently have.
 
-We will have remdiation that has object payload to be `KubletConfig`. As a exmaple below:
+We will have remediation that has object payload to be `KubletConfig`. As an example below:
```
Comment (JAORMX) [inline doc/kubeletConfig-remediations.md:14]: It's `KubeletConfig`, not `KubletConfig`

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: doc/kubeletConfig-remediations.md (hunk 1 above)
Comment (Vincent056) [reply doc/kubeletConfig-remediations.md:14]: Sorry for the typo

**Comment by Vincent056** [comment]
> If we're improving the documentation, can we also standardize on using `KubeletConfig` instead of `KubletConfig`? Otherwise looks good.

Sorry, I didn't see your comments earlier, I was meant to say`KubeletConfig` instead of  `KubletConfig`. Just fixed it !

**Review by JAORMX — APPROVED** [review]
/lgtm

**Pull request merged**